examples/custom-ipfs-repo/index.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const series = require('async/series')
+const IPFS = require('ipfs')
+
+// Create our custom options
+const customRepositoryOptions = {
+
+  /**
+   * Storage Backends are fully customizable. Each backend can be stored in seperate services,
+   * or in a single service. Options can be passed into the datastores via the storageBackendOptions
+   * property shown below.
+   */
+  storageBackends: {
+    root: require('datastore-fs'), // version and config data will be saved here
```

Comment:
This little description is super helpful. Maybe you could add something similar for the rest of these? Alternatively, maybe you could add some of this info to the docs for `js-ipfs-repo`.